package.json

```diff
@@ -15,7 +15,7 @@
     "lint:js": "eslint packages/*/src/ packages/*/*.stories.tsx utils/ .storybook/**/*.js --ext js,ts,tsx --max-warnings 0",
     "lint:md": "markdownlint README.md packages/*/src/**/*.md packages/*/src/*.md packages/*/README.md",
     "new": "utils/scripts/new.js",
-    "prepare": "yarn build",
+    "prepare": "husky install && yarn build",
```

Comment:
Let's move this to `postinstall`. There's no reason for us to run `husky install` on `publish`. See other migrated repos for details.